src/main/java/bio/terra/service/snapshot/flight/create/SnapshotCreateFlight.java

```diff
@@ -292,6 +295,16 @@ public SnapshotCreateFlight(FlightMap inputParameters, Object applicationContext
     // The IAM code contains retries, so we don't make a retry rule here.
     addStep(new SnapshotAuthzIamStep(iamClient, snapshotService, snapshotReq, userReq, snapshotId));
 
+    // add data access control groups to the snapshot
```

Comment:
Maybe also say why it's done here, e.g., "Now that the snapshot exists in Sam, we can ..."